src/include/meta.h

```diff
@@ -37,6 +38,7 @@
  * Optimize comparisons against the metafile URI, flag handles that reference the metadata file.
  */
 #define WT_IS_METADATA(dh) F_ISSET((dh), WT_DHANDLE_IS_METADATA)
+#define WT_IS_LAS(dh) (dh->name_hash == WT_LAS_URI_HASHED)
```

Comment:
What if there is a collision?

Reply:
Definitely presents an issue, however low probability.

Reply:
I don't think the probability is low enough to be ignored given mongodb is widely used. May be not in this PR, but we need to address that I think. Probably better to spend some time to use btree LAS flag instead.

Reply:
I think on btree open we could validate that the dhandle name is equal to the LAS uri using strcmp, I think that shouldn't present too much of a performance concern. At which point we set the btree LAS flag and can avoid comparing the hash. @bvpvamsikrishna what do you think?

Reply:
It seems like that issue I was referring to has gone away, either way I've updated the code in the diagnostic block to assert if the LAS file isn't where we expect it to be using strcmp. As such I've removed the hash comparison code and am only using the btree flag.